box-share-sdk/src/main/java/com/box/androidsdk/share/fragments/InviteCollaboratorsFragment.java

```diff
@@ -111,8 +122,12 @@ public boolean areCollaboratorsPresent() {
         return false;
     }
 
-    public void setInviteCollaboratorsListener(InviteCollaboratorsListener listener) {
-        mInviteCollaboratorsListener = listener;
+    @Override
+    public void onActivityCreated(@Nullable Bundle savedInstanceState) {
+        super.onActivityCreated(savedInstanceState);
+        // Attach the listener to view once createView is complete
+        mInviteCollaboratorsListener = ((InviteCollaboratorsListener) getActivity());
```

Comment:
Yea that is a fair point. I kept the structure of old code as it is since I don't fully understand how the interface was used. Lets talk to figure out a good design.